cmd/release-controller/sync_release.go

```diff
@@ -417,3 +418,44 @@ func (c *Controller) ensureReleaseMirrorJob(release *releasecontroller.Release,
 func releaseMirrorJobName(tagName string) string {
 	return fmt.Sprintf("%s-alternate-mirror", tagName)
 }
+
+// ensureRemoveTagJob creates a job to copy rc_payload__{version} to remove__rc_payload__{version} in quay.io
+func (c *Controller) ensureRemoveTagJob(payload *v1alpha1.ReleasePayload, release *releasecontroller.Release) (*batchv1.Job, error) {
+	if len(release.Config.AlternateImageRepository) == 0 || len(release.Config.AlternateImageRepositorySecretName) == 0 {
+		return nil, fmt.Errorf("alternate repository or secret not configured")
+	}
+
+	jobName := fmt.Sprintf("%s-remove-tag", payload.Name)
+	return c.ensureJob(jobName, nil, func() (*batchv1.Job, error) {
+		// Get cli image from mirror or config
+		cliImage := "registry.ci.openshift.org/ocp/4.16:cli"
+		if mirror, err := c.releaseLister.ImageStreams(release.Target.Namespace).Get(release.Target.Name); err == nil {
+			cliImage = fmt.Sprintf("%s:cli", mirror.Status.DockerImageRepository)
+		}
+		if len(release.Config.OverrideCLIImage) > 0 {
+			cliImage = release.Config.OverrideCLIImage
+		}
+
+		job, prefix := newReleaseJobBase(jobName, cliImage, release.Config.AlternateImageRepositorySecretName)
+
+		rcPayloadTag := fmt.Sprintf("rc_payload__%s", payload.Name)
```

Comment:
We currently do not create release tags named anything like: `rc_payload__{version}`, so this logic does not make any sense to me.

Reply:
we will be using similar [here](https://github.com/openshift/release/blob/master/hack/qci_registry_pruner.py)

Reply:
I'm confused...
Based on this logic, the calculated "fromImage" will be something of the form: ` rc_payload__4.22.0-0.nightly-2025-12-17-101159`
Which will be passed into `oc mirror <FROM> <TO>`, correct?
What I'm questioning is why you think the "FROM" image even exists with that naming convention?
AFAIK, the release-controller doesn't have any logic to create such a named release.